src/wp-includes/class-wp-block.php

```diff
@@ -139,6 +139,24 @@ public function __construct( $block, $available_context = array(), $registry = n
 		$this->block_type = $registry->get_registered( $this->name );
 
 		$this->available_context = $available_context;
+		$this->update_available_context( $block, $available_context );
+
+		if ( ! empty( $block['innerHTML'] ) ) {
+			$this->inner_html = $block['innerHTML'];
+		}
+
+		if ( ! empty( $block['innerContent'] ) ) {
+			$this->inner_content = $block['innerContent'];
+		}
+	}
+
+	public function update_available_context( $block, $available_context ) {
+		$this->context = array();
+		if ( null === $this->available_context ) {
```

Comment:
When would `$this->available_context` be `null`? I'm wondering since it's populated in the constructor with a parameter that's always an array.

Reply:
Yes it will never be `null` so i removed it in https://github.com/WordPress/wordpress-develop/pull/7522/commits/a6db0fdbc27cbd071fafa330967bc27e4b95bc41